rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -157,126 +117,203 @@ The two options supported in Rancher are:
+    lb_config:
+      port_rules:
+      - source_port: 80
+        target_port: 80
+        service: web1
+        protocol: http
+    health_check:
+      port: 42
+      interval: 2000
+      unhealthy_threshold: 3
+      healthy_threshold: 2
+      response_timeout: 2000
+  web1:
+    scale: 2
 ```
 
-#### Advanced Load Balancing (L7)
+#### Port rules
 
-We also support L7 load balancing with advanced routing options, which include using host headers, host paths and specific target ports. Advanced routing options use `labels` in the `docker-compose.yml` file.
+Port rules are defined in the `rancher-compose.yml`. Since port rules are defined individually, there may be multiple port rules defined for the same service. By default, Rancher will prioritize these port rules based on a specific priority ordering. If you would like to change the ordering of the prioritization, you can also set specific [priority](#priority) order of the rules.
 
+1. Hostname with no wildcards and URL
+2. Hostname with no wildcards
+3. Hostname with wildcards and URL
+4. Hostname with wildcards
+5. URL
+6. Default (no hostname, no URL)
 
-Here's the basic syntax for applying the label. You would only use this label if you were going to do some advanced routing rules. Advanced routing rules are optional and all of the fields  are optional as well. This syntax shows if you were to use all the options.
+##### Source Port
 
-Label Key | Label Value
----| ---
-`io.rancher.loadbalancer.target.<SERVICE_NAME>` | `<REQUEST_HOST>:<SOURCE_PORT>/<REQUEST_PATH>=<TARGET_PORT>`
+The source port is one of the ports exposed on the host (i.e. a port that is in the `docker-compose.yml`).
 
-<br>
+If you want to create internal load balancer, then the source port does not need to match any of the ports in the `docker-compose.yml` file.
 
-> **Note:** If you use a source port with the advanced routing options, the source port must also be listed in `ports` section of the `docker-compose.yml`
+##### Target Port
 
-##### Linking Services in a Different Stack
+The target port is the private port on the service. This port correlates to the port exposed on the image used to start your service.
 
-In the label for target services, `<SERVICE_NAME>` is the name of the service. If your service is in another stack, the `<SERVICE_NAME>` will need to include stack name using the following format `<STACK_NAME>/<SERVICE_NAME>`. The link to the target service of other stacks will be under `external_links`.
+##### Protocol
 
-##### Syntax of the Combination of all Optional Fields
+There are multiple protocol types that are supported in the Rancher HAProxy load balancer.
 
-Since the fields are optional, we support all combinations of the fields. Here is the syntax for all combinations using request host, source port, path and target port.
+* `http` - By default, if no protocol is set, the load balancer uses `http`.
+* `tcp`
+* `https`
+* `tls`
+* `sni`
+* `udp`
```

Comment:
we should mention that udp is not supported by haproxy provider. And that in the future, certain providers can support only the subset of the protocols exposed in the API